college/file_type_detector.py

```
@@ -0,0 +1,31 @@
+    if linter_command:
+        try:
+            subprocess.run([linter_command, file], check=True)
+            print(f"Linting {file} successful!")
+        except subprocess.CalledProcessError as e:
+            print(f"Linting {file} failed: {e}")
+    else:
+        print(f"No linter configured for {ext}")
+
+
+# Walk through files in the current directory and lint them
+for root, dirs, files in os.walk('.'):
```

Ruff

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)